x-pack/plugins/apm/server/lib/alerts/register_transaction_duration_anomaly_alert_type.ts

```diff
@@ -88,6 +73,7 @@ export function registerTransactionDurationAnomalyAlertType({
 
       const anomalySearchParams = {
         body: {
+          terminateAfter: 1,
```

Comment:
@ogupte I added `terminateAfter` to improve perf of this alert since we only need a single document. Do you agree this is fine?